pkg/services/sqlstore/permissions/dashboard.go

```diff
@@ -113,36 +133,42 @@ func (f AccessControlDashboardPermissionFilter) Where() (string, []interface{})
 	folderWildcards := accesscontrol.WildcardsFromPrefix(dashboards.ScopeFoldersPrefix)
 
 	filter, params := accesscontrol.UserRolesFilter(f.user.OrgID, f.user.UserID, f.user.Teams, accesscontrol.GetOrgRoles(f.user))
-	rolesFilter := "AND role_id IN(SELECT distinct id FROM role " + filter + ")"
+	rolesFilter := " AND role_id IN(SELECT distinct id FROM role " + filter + ") "
```

Comment:
this distinct will always make the query slower, is it possible to do without it?